.classpath

```diff
@@ -11,6 +15,51 @@
 	<classpathentry kind="lib" path="lib/nc.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/javax.mail.jar"/>
 	<classpathentry kind="lib" path="lib/bsh-core.jar"/>
+	<classpathentry kind="lib" path="lib.test/apiguardian-api.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-jupiter.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-jupiter-api.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-jupiter-engine.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-jupiter-params.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-platform-commons.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-platform-engine.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/junit-platform-launcher.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib.test/opentest4j.jar">
+		<attributes>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
```

Comment:
Ich würde hier `<classpathentry kind="lib" path="lib.test" including="*.jar">` verwenden. Da werden zwar alle jars des Artifacts "junit-bom" inkludiert; dass stört jedoch im classpath nicht. Dafür reduziert sich das dependency management auf die `pom.xml`.

Reply:
Hmm, ich verstehe gerade gar nicht, wie `opentest4j` durch resolve-dependencies heruntergeladen wird. Scheinbar wird doch die `.classpath` geparst.

Reply:
opentest4j ist eine transitive Abhängigkeit von junit
```
   org.junit.jupiter:junit-jupiter:jar:5.12.1:test
      org.junit.jupiter:junit-jupiter-api:jar:5.12.1:test
         org.opentest4j:opentest4j:jar:1.3.0:test
         org.junit.platform:junit-platform-commons:jar:1.12.1:test
```

Reply:
> Ich würde hier `<classpathentry kind="lib" path="lib.test" including="*.jar">` verwenden. Da werden zwar alle jars des Artifacts "junit-bom" inkludiert; dass stört jedoch im classpath nicht. Dafür reduziert sich das dependency management auf die `pom.xml`.

Entschuldige bitte! Was auf der Kommandozeile funktioniert, funktioniert nicht zwangsweise in Eclipse. Das kommt nämlich damit überhaupt nicht klar. Bitte mach das wieder rückgängig.

Wir können die `.classpath` später aufräumen und an maven binden, wie du es ohnehin schon vor hast.

Reply:
Brauchst es nicht revertieren, da diese jars hier gar nicht benötigt werden, wenn man den JUnit Container als Module dependency hinzugefügt.